README.md

```diff
@@ -101,9 +101,11 @@ Starting this `Debug configuration` will then connect to the process started via
 Execute `../gradlew docker` within org.testeditor.web.xtext.persistence
 
 ``` shell
-docker run -p 8080:8080 -e TARGET_REPO=https://github.com/test-editor/language-examples.git -e BRANCH_NAME=feature/gonk test-editor/xtext &
-docker run -p 9080:8080 -e GIT_PRIVATE_KEY="$(cat ~/.ssh/id_github_rsa)" -e KNOWN_HOSTS="$(cat ~/.ssh/known_hosts)" -e TARGET_REPO=git@github.com:test-editor/language-examples.git -e BRANCH_NAME=feature/gonk test-editor/persistence &
-
-GIT_PRIVATE_KEY="$(cat ~/.ssh/id_github_rsa)" KNOWN_HOSTS="$(cat ~/.ssh/known_hosts)" docker-compose up
+docker run -p 8080:8080 -e GIT_PRIVATE_KEY="$(cat ~/.ssh/id_github_rsa)" -e KNOWN_HOSTS_CONTENT="$(cat ~/.ssh/known_hosts)" -e TARGET_REPO=git@github.com:test-editor/language-examples.git -e BRANCH_NAME=feature/gonk testb-editor/xtext &
```

Comment:
In the example given for the config.yml files (see above) the branch given is `feature/some`. Maybe, for consistency, change it here, too?